src/openrct2-ui/windows/Footpath.cpp

```diff
@@ -888,7 +888,7 @@ static void window_footpath_place_path_at_point(int32_t x, int32_t y)
 
     // Try and place path
     gGameCommandErrorTitle = STR_CANT_BUILD_FOOTPATH_HERE;
-    auto footpathPlaceAction = FootpathPlaceAction({ x, y, z * 8 }, currentType, selectedType);
+    auto footpathPlaceAction = FootpathPlaceAction({ screenCoords.x, screenCoords.y, z * 8 }, currentType, selectedType);
```

Comment:
No game actions take screen cords. Any time you see something like this it is obvious something is wrong

Reply:
What to do? Build a `CoordsXY` or `TileCoordsXY`? I mean, they'd still be disguised screen coordinates.

Reply:
Well it's the same issue as above remove the assignments and use the map cord parameters directly.